src/components/Rewards.tsx

```diff
@@ -4,43 +4,70 @@ import { IRewardQueryOptions as FilterOptions } from "@daostack/client";
 import {
   Arc as Protocol,
   ArcConfig as ProtocolConfig,
-  ArcReward as Component,
+  InferredReward as Component,
   RewardEntity as Entity,
   RewardData as Data,
+  DAO,
+  DAOEntity,
+  Member,
+  MemberEntity,
+  Proposal,
+  ProposalEntity,
+  Token,
+  TokenEntity,
   CProps,
   ComponentList,
   ComponentListLogs,
   ComponentListProps,
+  applyScope,
 } from "../";
 import { CreateContextFeed } from "../runtime/ContextFeed";
 
-type RequiredProps = ComponentListProps<Entity, Data, FilterOptions>;
+type Scopes = "DAO" | "Member as beneficiary" | "Proposal" | "Token";
```

Comment:
Why are we using `Member as beneficiary` and not just member? (And same for every entity that has this)

I think it would be easier to just use `member`

Reply:
It's because a Member could be included multiple times in a schema. For example:
```graphql
type Proposal {
  ...
  proposer: Member
  beneficiary: Member
}
```